### DISCUSSION TITLE
Fastapi performance issue not responding  important

### DISCUSSION THREAD

**Discussion by iiotsrc** [fastapi/fastapi#12964] [Questions]
### Privileged issue

- [X] I'm @tiangolo or he asked me directly to create an issue here.

### Issue Content

Hai 
We using fastapi last 1.5 Yr we facing daily issues on that on performance like 100 to 200 clients requesting every 5 sec some time not responding some time totally hanging we restarting the services 
Additional information we used mysql

**Comment by YuriiMotov** [comment]
It's impossible to answer your question until you provide more details.
Can you create a MRE that reproduces your problem?

**Comment by estebanx64** [comment]
Hi there!

It is really hard to know what is happening with the details you are providing.

We need for example what is the architecture, are you using uvicorn workers directly or not? Is your database sync or async?
Do you have another code that is blocking the progress?
Do you have load balancers in front of them?

We need a very detailed report to figure out if the problem is something related to fastapi or if it's an architectural design problem 🤓

**Comment by YuriiMotov** [comment]
Closing this. Feel free to reopen it when you are ready to provide more info
